src/main/java/org/testeditor/fixture/web/WebDriverFixture.aml

```diff
@@ -0,0 +1,148 @@
+package org.testeditor.fixture.web
+
+interaction type wait {
+	label = "Wait seconds"
+	template = "Wait" ${seconds} "seconds"
+	method = WebDriverFixture.waitSeconds(seconds)
+}
+
+interaction type startBrowser {
+	label = "Start browser"
```

Comment:
the other labels ca be removed i think. there is no new infomation then the type or dsl